src/main/java/bio/terra/service/auth/ras/RasDbgapPermissions.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.service.auth.ras;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record RasDbgapPermissions(String consent_group, String phs_id) {}
```

Comment:
I think Jackson can handle camel case mapping for snake case names automatically. Does it work if you use camel case here?
```suggestion
public record RasDbgapPermissions(String consentGroup, String phsId) {}
```

Camel case would be better if possible, to follow our coding conventions.

Reply:
I can test it out!

Reply:
I tested this, and it does not work out of the box.  I tried adding the following notation with the camel case record elements, that also didn't work:

```
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

The error message reflected a known incompatibility between `@JsonNaming` notations and Java records: https://github.com/FasterXML/jackson-databind/issues/3102

I talked with @tlangs and Identiteam may end up defining this structure in the ECM client / a common library, so I'll leave this record as it is for now.